mongo.go

```diff
@@ -201,9 +201,26 @@ func (m *Handler) Find(ctx context.Context, lookup *resource.Lookup, page, perPa
 	defer m.close(c)
 	var mItem mongoItem
 	query := c.Find(q).Sort(s...)
+
+	// Perform request the first time (to get total number of results without any filter applied)
+	iter := query.Iter()
+	tmp := &resource.ItemList{Page: page, Offset: skip, Total: 0, Items: []*resource.Item{}}
+	for iter.Next(&mItem) {
+		// Check if context is still ok before to continue
+		if err = ctx.Err(); err != nil {
+			// TODO bench this as net/context is using mutex under the hood
+			iter.Close()
+			return nil, err
+		}
+		tmp.Items = append(tmp.Items, newItem(&mItem))
+	}
+
```

Comment:
Please remove that from the PR. Most people don't want this info, and it is not acceptable to double the number of requests for any given `Find`.